### PR TITLE
update: container registry

### PR DESCRIPTION
**Pull request by gam4it**
- replace `docker.pkg.github.com` with `ghcr.io` due to https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-docker-registry

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,16 +116,19 @@
 You will encounter the following log message in your GitHub Actions Pipelines:
 
 ```
-WARNING! Using --password via the CLI is insecure. Use --password-stdin.
+WARNING! Using -*** the CLI is insecure. Use --password-stdin.
```
Comment (VaultVulp) [inline README.md]: What for?

Comment (gam4it) [reply README.md]: I just copy & paste what I see from the output which is a little bit different from what you saw before.

**Comment by asksmruti** [comment]
Could you please merge this PR as this action is failing and it's not usable with current version ?

**Comment by asksmruti** [comment]
Anyway, it's fine. someone already has published another package with this fix.

File: README.md (hunk 1 above)
Comment (VaultVulp) [reply README.md]: Could you please restore original state of this line, for it makes this message rather hard to understand.

**Review by VaultVulp — APPROVED** [review]

**Comment by VaultVulp** [comment]
Thanks for you contribution, @gam4it ! Have a great day! I'll push a new release, ASAP.

**Pull request merged**